src/mon/FSCommands.cc

```diff
@@ -448,6 +448,18 @@ class SetHandler : public FileSystemCommandHandler
       {
         fs.get_mds_map().set_max_filesize(n);
       });
+    } else if (var == "snap_rstat") {
+      bool snap_rstat = true;
+      int r = parse_bool(val, &snap_rstat, ss);
+      if (r != 0) {
+        return r;
+      }
+      fsmap.modify_filesystem(
+          fs->fscid,
+          [snap_rstat](std::shared_ptr<Filesystem> fs)
+      {
+        fs->mds_map.set_snap_rstat(snap_rstat);
```

Comment:
`mds_map` is private, `get_mds_map()` needs to be called instead

```suggestion
          [snap_rstat](auto&& fsp)
      {
        fsp.get_mds_map().set_snap_rstat(snap_rstat);
```

Reply:
@rishabh-d-dave Done and fixed them both. Thanks!